jupyter_server/base/handlers.py

```diff
@@ -11,6 +11,7 @@
 import mimetypes
 import os
 import re
+import sqlite3
```

Comment:
Should there be a guard in case if Python build does no include `sqlite3`? There is one here:

https://github.com/jupyter-server/jupyter_server/blob/f37967ba7472d0cab4afeda3cc946ac244a3164e/jupyter_server/services/sessions/sessionmanager.py#L13-L17

Reply:
@krassowski 

Good catch, I also saw that these attributes were only introduced in python 3.11, so I've made this check a bit more robust, with a string based comparison as fallback